cmd/root.go

```diff
@@ -33,26 +32,9 @@ var opts *flags.Root
 
 func CreateConfig() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
-	var cfgFile string
-
-	flag.StringVar(&cfgFile, "config", "", "Config file to be used. If empty configs will be read from cmd.")
 	opts = &flags.Root{}
 	opts.InitFlags()
 	flag.Parse()
-
-	if cfgFile != "" {
```

Comment:
We could remove all the struct tags now that we drop support for files.
That is, all the `json:"..."` from [files](https://github.com/ExpediaGroup/mittens/blob/master/cmd/flags/file_probe.go#L23) where we define flags.